src/tools/yarprobotstatepublisher/src/robotstatepublisher.cpp

```diff
@@ -255,16 +280,51 @@ void YARPRobotStatePublisherModule::onRead(yarp::rosmsg::sensor_msgs::JointState
         sizeOfTFFrames = model.getNrOfFrames();
     }
 
-    for (size_t frameIdx=0; frameIdx < sizeOfTFFrames; frameIdx++)
-    {
-        if(m_baseFrameIndex == frameIdx)    // skip self-tranform
-            continue;
-
-        iDynTree::Transform base_H_frame = m_kinDynComp.getRelativeTransform(m_baseFrameIndex, frameIdx);
-        iDynTree::toYarp(base_H_frame.asHomogeneousTransform(), m_buf4x4);
-        m_iframetrans->setTransform(m_tfPrefix + model.getFrameName(frameIdx),
-                                    m_tfPrefix + model.getFrameName(m_baseFrameIndex),
-                                    m_buf4x4);
+    // Publish the frames on TF
+    bool m_publishGlobalTF = false; // TODO
```

Comment:
Just to understand, why does this part of the code change? Is this related to the `model-joints` parameter?

Reply:
My bad, I didn't resolve the conflicts correctly after merging #548 to devel!  
Updated in https://github.com/robotology/idyntree/pull/533/commits/fd6f66f37697ba5ef6d524b825afb076c6cfd4e5